### PR TITLE
Added Armstrong number in dart

### PR DESCRIPTION
**Pull request by ritvij14**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

This code checks if the number input is Armstrong or not. https://replit.com/@ritvij14/DSA-NeoAlgo#armstrong.dart

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
#3098

### CODE REVIEW

**Comment by ritvij14** [comment]
@ankitaggarwal23 @siddharth25pandey please review this.

File: Dart/maths/Armstrong.dart [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+ * 
+ * Enter Number: 
+ * 153
+ * 153 is an Armstrong Number
+ * 
+ * Enter Number: 
+ * 224
+ * 224 is not an Armstrong Number
+ */
+
```
Comment (HarshCasper) [inline Dart/maths/Armstrong.dart]: Remove extra lines and add a new line at the EOF

File: Dart/maths/Armstrong.dart [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import 'dart:math';
+import 'dart:io';
+
+bool isArmstrong(int num) {
```
Comment (HarshCasper) [inline Dart/maths/Armstrong.dart:5]: Add function description

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: Dart/maths/Armstrong.dart (hunk 1 above)
Comment (ritvij14) [reply Dart/maths/Armstrong.dart]: Removed extra lines, added new line at EOF

File: Dart/maths/Armstrong.dart (hunk 2 above)
Comment (ritvij14) [reply Dart/maths/Armstrong.dart:5]: @HarshCasper done

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**